sdcm/tester.py

```diff
@@ -2049,6 +2056,28 @@ def get_truncated_time_from_system_truncated(self, session, table_id):  # pylint
         truncated_time = self.rows_to_list(session.execute(query))
         return truncated_time[0]
 
+    def get_describecluster_info(self) -> Optional[ClusterInformation]:
+        node = self.db_cluster.get_node()
+
+        try:
```

Comment:
let's not handle the exception here and let the test crash when there is an issue in nodetool command or in parsing logic